language/en/INGAME.php

```diff
@@ -56,8 +57,8 @@
 
 $LNG['page_doesnt_exist']					= 'This page does not exist';
 $LNG['bad_forum_url']				    	= 'The Forum link is incorrect!';
-$LNG['admin_access_1']				    	= 'Players have access temporarily s administrative functions!';
-$LNG['admin_access_link']				    = 'Disable this access';
+$LNG['admin_access_1']				    	= 'Players have temporary administrative permissions.';
+$LNG['admin_access_link']				    = 'Access Denied';
```

Comment:
'Disable access'/'Deny access'? This is action

Reply:
Misunderstood the intention of the original heavily broken sentence. The closest thing I could come up with is that it was trying to say access is denied; based on its own self-consistence broken wording. I get what you mean, so the line should be changed to simply "disable". The line above it is as it should be.